src/librustdoc/clean/types.rs

```diff
@@ -125,6 +124,20 @@ crate struct Crate {
     crate collapsed: bool,
 }
 
+// `Crate` is frequently moved by-value. Make sure it doesn't unintentionally get bigger.
+#[cfg(all(target_arch = "x86_64", target_pointer_width = "64"))]
+rustc_data_structures::static_assert_size!(Crate, 104);
+
+impl Crate {
+    crate fn name(&self, tcx: TyCtxt<'_>) -> Symbol {
+        ExternalCrate::LOCAL.name(tcx)
+    }
+
+    crate fn src_file_name(&self, tcx: TyCtxt<'_>) -> FileName {
```

Comment:
nit: I would rather name this `src` to be consistent with `ExternalCrate`

Reply:
Yeah, consistency is probably best here. `src` is just a bit of a confusing name ;)

I renamed it back.